department-of-veterans-affairs/abd-vro
# Mcp 1899 - Enable spotlessCheck for service-data-access

Enable spotlessCheck for service-data-access

## Description

service-data-access build process did not run spotless check. This PR enables the spotless check for service-data-access sub project.

### What was the problem?

The Gradle build did not set to run spotless check for service-data-access.

### How does this fix it?

Enable the spotless check.

### Jira Tickets



- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How to test this PR

- Step 1. Build the application.
- Step 2. Observe the spotlessJavaCheck result for service-data-access. 

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
